docs/source/installation.rst

```diff
@@ -108,6 +108,9 @@ uses export to set variables):
     export MESASDK_ROOT=/Applications/mesasdk
     source $MESASDK_ROOT/bin/mesasdk_init.sh
 
+    # add shmesa (the MESA command line tool) to your PATH 
+    PATH=$PATH:$MESA_DIR/scripts/shmesa
```

Comment:
I think we want an `export` here to get the path update into the user's environment.

Reply:
I thought $PATH is exported by default (so doesn't need to be re-exported). Nevertheless I've added it as there's no harm.